operations/operations/publishing/publishing-guidelines/writing-an-mvp.md

```diff
@@ -0,0 +1,40 @@
+# Writing and submitting documentation with your PR
+
+Our user, admin, and technical documentation lives at [docs.mattermost.com](https://docs.mattermost.com), primarily written using ReStructured Text, using Sphinx and Write The Docs as our build and theme tools. 
+
+If your PR is for a new feature, major update to an existing feature, or a fix that impacts the current documentation, it's recommended that you include documentation around the change. This is helpful in providing context for reviewers, and also expedites the content being added to the technical documentation. 
```

Comment:
Maybe in places where we refer to "technical documentation" we say docs.mattermost.com instead. I'm 0/5 and I'm not sure what is the best way to refer to the docs site.

On https://developers.mattermost.com/ at the top far-right we refer to it as "Admin docs".

Reply:
Done. :)